libs/shared/ui/src/lib/styles/components/select-multiple.scss

```diff
@@ -0,0 +1,82 @@
+.input--select-multiple__control {
```

Comment:
Adding a comment on the top to explain the use of !important 

And the reset can be done with Tailwind?

Reply:
You can't set important with Tailwind in css file, if I put a class like this `!text-red` it return an error